mlir/test/Dialect/Tensor/fold-into-pack-and-unpack.mlir

```diff
@@ -114,3 +114,102 @@ func.func @pad_pack_different_padding_value(%src: tensor<16641x16xf32>) -> tenso
 // CHECK-LABEL: func.func @pad_pack_different_padding_value
 // CHECK:         tensor.pad
 // CHECK:         tensor.pack
+
+// -----
+
+func.func @tensor_pack_linalg_transpose_fold(%arg0: tensor<56x57x1x64xf32>) -> tensor<1x57x56x2x32xf32> {
```

Comment:
Can we please also add a test with dynamic dimensions and tiles?

Reply:
Here is an example about dynamic inner tiles:

https://github.com/llvm/llvm-project/blob/f2afd10776bda7e0dc23176afd1a99d4ccbf967c/mlir/test/Dialect/Tensor/ops.mlir#L389-L400